torch_xla/distributed/fsdp/state_dict_utils.py

```diff
@@ -22,7 +22,11 @@ def _consolidate_param(state_dict_list, shard_metadata, name, prefix, suffix):
     orig_size = p_info["_orig_size"]
 
   full_param = torch.cat(p_shard_list, dim=0)
-  full_param = full_param[:_numel(orig_size)].view(*orig_size)
+  try:
+    full_param = full_param[:_numel(orig_size)].view(*orig_size)
+  except RuntimeError:
```

Comment:
do you mind expand a bit more on this line?

Reply:
Thanks, I revised it to an if-statement checking `full_param.dim() == 1` in the new commit below.